### PR TITLE
Dev

### PR DESCRIPTION
**Pull request by VladimirNalivaiko**

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -0,0 +1,163 @@
...
+              Code with confidence – Spring
+               provides an open programming
+                model that is comprehensive,
+                 cohesive, widely understood and well-supported.
+            </div>
+          </div>
+        </div>
+      </div>
+    </section>
+    <footer class="footer">
```
Comment (paveldz) [inline src/index.html]: Is class="footer" required?

File: src/Content/css/responsive-style.css [hunk 2]
```diff
@@ -0,0 +1,255 @@
+@media (max-width: 700px) {
```
Comment (paveldz) [inline src/Content/css/responsive-style.css]: Spaces and tabs are mixed. Why did this happen?

File: src/index.html [hunk 3]
```diff
@@ -0,0 +1,163 @@
...
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <meta http-equiv="X-UA-Compatible" content="ie=edge">
+  <link rel="stylesheet" href="Content/css/style.css" />
+  <link rel="stylesheet" href="Content/css/responsive-style.css" />
+  <link rel="stylesheet" href="Content/css/font-awesome.css" />
+  <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/4.0.0-beta/css/bootstrap.min.css"
+  integrity="sha384-/Y6pD6FV/Vv2HJnA6t+vslU6fwYXjCFtcEpHbNJ0lyAFsXTsjBbfaDjzALeQsN6M"
+  crossorigin="anonymous">
+  <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.1.1/jquery.min.js" type="text/javascript" ></script>
+  <script src="js/main.js" type="text/javascript"></script>
```
Comment (paveldz) [inline src/index.html]: Is it possible to load these scripts after document rendering?

File: src/index.html [hunk 4]
```diff
@@ -0,0 +1,163 @@
...
+  <title>Spring</title>
+</head>
+<body>
+  <div class="navigation">
```
Comment (paveldz) [inline src/index.html]: Please use semantic tags where possible

File: src/index.html [hunk 5]
```diff
@@ -0,0 +1,163 @@
...
+        </div>
+        <ul class="nav-list">
+          <li class="link"><a href="#">DOCS</a></li>
+          <li class="link"><a href="#">GUIDES</a></li>
+          <li class="link"><a href="#">PROJECTS</a></li>
+          <li class="link"><a href="#">BLOG</a></li>
+          <li class="link"><a href="#">QUESTIONS</a></li>
+          <li class="link">
+             <i class="fa fa-search fa-2x" id="search-icon"
+               onclick="searchClick()" aria-hidden="true"></i>
```
Comment (paveldz) [inline src/index.html]: Is aria-hidden attribute required here?

File: src/index.html [hunk 6]
```diff
@@ -0,0 +1,163 @@
...
+  <div class="sidebar">
+    <ul class="mobile-nav-list">
+      <li class="mobile-nav-search">
+        <div class="search-container">
+          <input type="text" placeholder="Search..." class="mobile-input">
+          <button class="modile-search-icon fa fa-search"></button>
+        </div>
+      </li>
+      <li class="mobile-nav-list-item">
+        <a id="mobile-nav-item-home" href="">Home<i class="fa fa-chevron-right" aria-hidden="true"></i></a>
```
Comment (paveldz) [inline src/index.html]: Is aria-hidden attribute required here?

Comment (paveldz) [reply src/index.html]: And below

File: src/index.html [hunk 7]
```diff
@@ -0,0 +1,163 @@
...
+      <li class="mobile-nav-list-item">
+        <a id="mobile-nav-item-blog" href="">Blog<i class="fa fa-chevron-right" aria-hidden="true"></i></a>
+      </li>
+      <li class="mobile-nav-list-item">
+        <a id="mobile-nav-item-questions" href="">Questions<i class="fa fa-chevron-right" aria-hidden="true"></i></a>
+      </li>
+    </ul>
+  </div>
+  <div class="content">
+    <section class="meeting-wrapper">
```
Comment (paveldz) [inline src/index.html]: meeting? Why meeting? :) It's not clear for me

File: src/index.html [hunk 8]
```diff
@@ -0,0 +1,163 @@
...
+      <div class="meeting-icon">
+        <i class="fiture-certification"></i>
```
Comment (paveldz) [inline src/index.html]: Is **i** tag is required here?

File: src/index.html [hunk 9]
```diff
@@ -0,0 +1,163 @@
...
+          Let's build a better Enterprise.
+        </div>
+        <br>
+        <div class="meeting-info">
+          Spring helps development teams everywhere build simple, portable,<br>
+          fast and flexible JVM-based systems and applications.
+        </div>
+      </div>
+    </section>
+    <section class="container-fluid advantages">
```
Comment (paveldz) [inline src/index.html:89]: This is only one occurrence of container-fluid class. Looks like this is copy-paste from spring.io

File: src/index.html (hunk 8 above)
Comment (paveldz) [reply src/index.html]: class="fiture-certification"
fiture? Typo?
Looks like this is copy-past from spring.io

File: src/index.html (hunk 5 above)
Comment (paveldz) [reply src/index.html]: Please move adding of event handler to js. You are mixed few approaches (There is one more adding of event handler in js)

File: src/index.html [hunk 10]
```diff
@@ -0,0 +1,163 @@
...
+          <a href=""></a>
+        </div>
+        <ul class="nav-list">
+          <li class="link"><a href="#">DOCS</a></li>
+          <li class="link"><a href="#">GUIDES</a></li>
+          <li class="link"><a href="#">PROJECTS</a></li>
+          <li class="link"><a href="#">BLOG</a></li>
+          <li class="link"><a href="#">QUESTIONS</a></li>
+          <li class="link">
+             <i class="fa fa-search fa-2x" id="search-icon"
```
Comment (paveldz) [inline src/index.html]: Are fa-* classes required?

File: src/Content/css/font-awesome.css [hunk 11]
```diff
@@ -0,0 +1,2337 @@
+/*!
```
Comment (paveldz) [inline src/Content/css/font-awesome.css:1]: Why presented two versions of font-awesome? (minimized and not minimized).
Are these styles required at all?

File: src/Content/css/responsive-style.css [hunk 12]
```diff
@@ -0,0 +1,255 @@
...
+		text-align: center;
+		border: 0;
+		display: block;
+	}
+
+	.fiture-certification{
+		display: none;
+	}
+
+	.meeting-title{
```
Comment (paveldz) [inline src/Content/css/responsive-style.css]: Styles for this class are presented few times. Some of them are overridden. Why?

File: src/Content/css/responsive-style.css [hunk 13]
```diff
@@ -0,0 +1,255 @@
...
+  }
+	.navigation	{
+		height: 50px;
+		position: relative;
+		text-align: center;
+		border: 0;
+		display: block;
+	}
+
+	.fiture-certification{
```
Comment (paveldz) [inline src/Content/css/responsive-style.css]: Styles for this class are presented few times. Why?

File: src/js/main.js [hunk 14]
```diff
@@ -0,0 +1,31 @@
...
+  });
+
+  setActiveLink();
+});
+
+var searchClick = function(event) {
+  $('.search-box').toggleClass('active');
+  $('#search-icon').toggleClass('active');
+
+  var icon = document.getElementById('search-icon');
```
Comment (paveldz) [inline src/js/main.js]: Why did you mix plain js and jQuery? Please use jQuery

File: src/js/main.js [hunk 15]
```diff
@@ -0,0 +1,31 @@
+jQuery(document).ready(function($) {
+  $('.burger-btn').click(function(event) {
+    $('.sidebar').toggleClass('active');
+    $('.navigation').toggleClass('active');
+    $('.content').toggleClass('active');
+  });
+
+  $('.mobile-nav-list-item').click(function(event) {
+    localStorage.setItem('clicked-link', event.target.id);
```
Comment (paveldz) [inline src/js/main.js:9]: Local storage is redundant here. You can just set appropriate class, setActiveLink function does it

File: src/Content/css/responsive-style.css [hunk 16]
```diff
@@ -0,0 +1,255 @@
...
+		width: 100%;
+	}
+
+	.meeting-block{
+		width: 89%;
+		margin: -20px auto 0 auto;
+	}
+}
+
+.sidebar{
```
Comment (paveldz) [inline src/Content/css/responsive-style.css]: Styles for this class are presented few times. Why?

Comment (paveldz) [reply src/Content/css/responsive-style.css]: Please also review all defined classes

File: src/index.html [hunk 17]
```diff
@@ -0,0 +1,163 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <meta http-equiv="X-UA-Compatible" content="ie=edge">
```
Comment (paveldz) [inline src/index.html]: Is it required?

**Review by paveldz — CHANGES_REQUESTED** [review]

File: src/index.html (hunk 17 above)
Comment (VladimirNalivaiko) [reply src/index.html]: No, the 6'th line isn't required. This line pasted here by emmet plugin.

File: src/index.html (hunk 3 above)
Comment (VladimirNalivaiko) [reply src/index.html]: Yes

File: src/index.html (hunk 10 above)
Comment (VladimirNalivaiko) [reply src/index.html]: Yes, they are required, because i use font-awesome

File: src/index.html (hunk 5 above)
Comment (VladimirNalivaiko) [reply src/index.html]: sure no, okay

File: src/index.html (hunk 6 above)
Comment (VladimirNalivaiko) [reply src/index.html]: okay

File: src/index.html (hunk 7 above)
Comment (VladimirNalivaiko) [reply src/index.html]: it's quite hard for me to crete classes names. I thougth that this div is the first div, that users will see on this page. And it's the reason, why i name this div that way

File: src/index.html (hunk 8 above)
Comment (VladimirNalivaiko) [reply src/index.html]: Maybe **i** tag isn't the best way to render this picture.
It's only looks like it. I've downloaded pictures from spring.io and "feature-certifiacation"(yes, it's typo, i've corrected class name) was the name of downloaded picture

File: src/index.html (hunk 9 above)
Comment (VladimirNalivaiko) [reply src/index.html:89]: Here i used bootstrap class "container-fluid" to fill the full width of screen. it's not a copy-paste

File: src/index.html (hunk 1 above)
Comment (VladimirNalivaiko) [reply src/index.html]: You are right, it's redundant

File: src/Content/css/responsive-style.css (hunk 13 above)
Comment (VladimirNalivaiko) [reply src/Content/css/responsive-style.css]: accidentally posted it

File: src/Content/css/responsive-style.css (hunk 16 above)
Comment (VladimirNalivaiko) [reply src/Content/css/responsive-style.css]: reviewed

File: src/js/main.js (hunk 14 above)
Comment (VladimirNalivaiko) [reply src/js/main.js]: corrected

File: src/js/main.js (hunk 15 above)
Comment (VladimirNalivaiko) [reply src/js/main.js:9]: I don't understand, how i can get id of clicked element of after page reload. I think that locale storage is the best way to get it. Maybe i am wrong. Do you any better ideas?

File: src/Content/css/responsive-style.css (hunk 12 above)
Comment (VladimirNalivaiko) [reply src/Content/css/responsive-style.css]: it's my inattention. Corrected

File: src/Content/css/responsive-style.css (hunk 2 above)
Comment (VladimirNalivaiko) [reply src/Content/css/responsive-style.css]: corrected

File: src/Content/css/font-awesome.css (hunk 11 above)
Comment (VladimirNalivaiko) [reply src/Content/css/font-awesome.css:1]: i just wanted to use font-awesome. If it's wrong, i will correct it

File: src/js/main.js [hunk 18]
```diff
@@ -0,0 +1,28 @@
...
+    $('#search-icon').toggleClass('fa-search');
+    $('#search-icon').toggleClass('fa-times');
+    $('#search-icon').toggleClass('fa-2x');
+  });
+
+  setActiveLink();
+});
+
+var setActiveLink = function() {
+  var el = document.getElementById(localStorage.getItem('clicked-link'));
```
Comment (paveldz) [inline src/js/main.js]: plain js. please use jquery

**Review by paveldz — APPROVED** [review]

**Pull request merged**